test/DssVestTopUp.test.ts

```diff
@@ -177,7 +177,9 @@ describe("DssVestTopUp", function () {
           ["address", "uint256"],
           joinEvent?.data || ""
         );
+        const tokensInVowAddr = await token.balanceOf(vowAddr);
 
+        expect(tokensInVowAddr).to.equal(vestedTokens.sub(maxDepositAmt));
```

Comment:
This is assuming all vested tokens are accrued at this point. 

Also, if you're changing the approach why is this assertion not changed to use `vestedTokens`?

`expect(joinAmt).to.eq(unpaid.sub(maxDepositAmt));`

Reply:
Fair point but we are going to have some repeating `expect` functions. 

Therefore I think removing the event in the DaiJoinMock contract is going to be beneficial as we can do the same when we check the balance of the vow contract address.